<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page402</class>
 <widget class="QWidget" name="page402">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>272</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">page402</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#page402
{
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderPanel" name="headerPanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>72</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>36</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="ATCMbutton" name="atcmButton_next">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>ATCMbutton widget</string>
       </property>
       <property name="pageName">
        <string notr="true"/>
       </property>
       <property name="remember">
        <bool>false</bool>
       </property>
       <property name="statusvar">
        <string notr="true"/>
       </property>
       <property name="viewStatus">
        <bool>false</bool>
       </property>
       <property name="visibilityVar">
        <string notr="true"/>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="bgPressColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontPressColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pressText">
        <string/>
       </property>
       <property name="borderWidth">
        <number>0</number>
       </property>
       <property name="borderRadius">
        <number>0</number>
       </property>
       <property name="apparence">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" rowspan="3">
      <widget class="ATCMbutton" name="atcmButton_prev">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>ATCMbutton widget</string>
       </property>
       <property name="pageName">
        <string notr="true"/>
       </property>
       <property name="remember">
        <bool>false</bool>
       </property>
       <property name="statusvar">
        <string notr="true"/>
       </property>
       <property name="viewStatus">
        <bool>false</bool>
       </property>
       <property name="visibilityVar">
        <string notr="true"/>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="bgPressColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontPressColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pressText">
        <string/>
       </property>
       <property name="borderWidth">
        <number>0</number>
       </property>
       <property name="borderRadius">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_wattmeterN">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>M</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="ATCMlabel" name="atcmLabel_M_W">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>17</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>xxxx</string>
       </property>
       <property name="toolTip">
        <string>PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="whatsThis">
        <string>ATCMlabel widget</string>
       </property>
       <property name="variable">
        <string>PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="prefix">
        <string>M </string>
       </property>
       <property name="suffix">
        <string> W</string>
       </property>
       <property name="refresh">
        <number>200</number>
       </property>
       <property name="max">
        <string notr="true">0</string>
       </property>
       <property name="min">
        <string notr="true">0</string>
       </property>
       <property name="viewStatus">
        <bool>false</bool>
       </property>
       <property name="visibilityVar">
        <string/>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontColor">
        <color>
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="bgSelectColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontSelectColor">
        <color>
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="borderWidth">
        <number>1</number>
       </property>
       <property name="borderRadius">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ATCMlabel" name="atcmLabel_W">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>17</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>xxxx</string>
       </property>
       <property name="toolTip">
        <string>PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="whatsThis">
        <string>ATCMlabel widget</string>
       </property>
       <property name="variable">
        <string>PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="prefix">
        <string>M </string>
       </property>
       <property name="suffix">
        <string> W</string>
       </property>
       <property name="refresh">
        <number>200</number>
       </property>
       <property name="max">
        <string notr="true">0</string>
       </property>
       <property name="min">
        <string notr="true">0</string>
       </property>
       <property name="viewStatus">
        <bool>false</bool>
       </property>
       <property name="visibilityVar">
        <string/>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="bgSelectColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontSelectColor">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="borderWidth">
        <number>1</number>
       </property>
       <property name="borderRadius">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="6" colspan="3">
      <widget class="ATCMgraph" name="atcmGraph">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>175</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 170, 0);</string>
       </property>
       <property name="toolTip">
        <string>Y1=PLC_EP_wattmeter_M_W
Y2=PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="whatsThis">
        <string>ATCMgraph widget</string>
       </property>
       <property name="x1Variable">
        <string>PLC_time_bis</string>
       </property>
       <property name="x1Min">
        <string>PLC_timeMin_bis</string>
       </property>
       <property name="x1Max">
        <string>PLC_timeMax_bis</string>
       </property>
       <property name="x1Step">
        <double>60.000000000000000</double>
       </property>
       <property name="x1TickLabelVisible">
        <bool>true</bool>
       </property>
       <property name="x1TickLabelFormat">
        <enum>ATCMgraph::date</enum>
       </property>
       <property name="x1Label">
        <string/>
       </property>
       <property name="y1Variable">
        <string>PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="y1Min">
        <string>-4500</string>
       </property>
       <property name="y1Max">
        <string>4500</string>
       </property>
       <property name="y1Step">
        <double>500.000000000000000</double>
       </property>
       <property name="y1TickLabelVisible">
        <bool>true</bool>
       </property>
       <property name="y1Label">
        <string>W</string>
       </property>
       <property name="pen1Color">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="Display1">
        <string/>
       </property>
       <property name="x2Variable">
        <string>PLC_time_bis</string>
       </property>
       <property name="x2Min">
        <string>PLC_timeMin_bis</string>
       </property>
       <property name="x2Max">
        <string>PLC_timeMax_bis</string>
       </property>
       <property name="x2Step">
        <double>10.000000000000000</double>
       </property>
       <property name="x2Label">
        <string/>
       </property>
       <property name="y2Variable">
        <string>PLC_EP_wattmeter_M_W</string>
       </property>
       <property name="y2Min">
        <string>-4500</string>
       </property>
       <property name="y2Max">
        <string>4500</string>
       </property>
       <property name="y2Step">
        <double>500.000000000000000</double>
       </property>
       <property name="y2TickLabelVisible">
        <bool>true</bool>
       </property>
       <property name="y2Label">
        <string>W</string>
       </property>
       <property name="pen2Color">
        <color>
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="Display2">
        <string/>
       </property>
       <property name="gridVisible">
        <bool>false</bool>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCMbutton</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMgraph</class>
   <extends>QFrame</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>ATCMlabel</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderPanel</class>
   <extends>QWidget</extends>
   <header>headerpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>atcmButton_next</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page402</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_page(const char*,bool)</slot>
 </slots>
</ui>
